src/css/styles.css

```diff
@@ -0,0 +1,126 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat&display=swap");
+
+@import url("./reset.css");
+@import url("./variables.css");
+
+html,
+body {
+  font-family: var(--t-font-family);
+  font-size: var(--t-font-size);
+}
+
+h1,
+.is-h1 {
+  font-size: var(--t-font-size-xxlg);
+  line-height: var(--t-line-height-xxlg);
+  font-weight: var(--t-font-weight-semibold);
+}
+
+h2,
+.is-h2 {
+  font-size: var(--t-font-size-xlg);
+  line-height: var(--t-line-height-xlg);
+  font-weight: var(--t-font-weight-semibold);
+}
+
+h3,
+.is-h3 {
+  font-size: var(--t-font-size-lg);
+  line-height: var(--t-line-height-xlg);
+  font-weight: var(--t-font-weight-semibold);
+}
+
+h4,
+.is-h4 {
+  font-size: var(--t-font-size-md);
+  line-height: var(--t-line-height-md);
+  font-weight: var(--t-font-weight-semibold);
+}
+
+h5,
+.is-h5 {
+  font-size: var(--t-font-size-sm);
+  line-height: var(--t-line-height-sm);
+  font-weight: var(--t-font-weight-semibold);
+}
+
+h6,
+.is-h6 {
+  font-size: var(--t-font-size-xsm);
+  line-height: var(--t-line-height-xsm);
+  font-weight: var(--t-font-weight-semibold);
+}
+
+p {
+  font-size: var(--t-font-size-md);
+  line-height: var(--t-line-height-md);
+  font-weight: var(--t-font-weight-regular);
+}
+
+/*
+ * Helpers
+ * ~ TOC ~
+ * 1.  Font Size
+ * 2.  Line Height
+ * 3.  Font Weight
+ */
```

Comment:
Remove the commented css

Reply:
This is a comment that I have provided on purpose because once this CSS file becomes long it would b easy to navigate and find things in the CSS fine. This comment is supposed to work like a table of content that we have at the beginning of a book to help us navigate around easily. (Suggestion By Internet.)

This is especially useful because we are using CSS. if we were to use SCSS this would not be necessary as the folder structure will be segregated based on components and would-be helpful to maintain code.